src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -692,4 +693,28 @@ private function addCacheSection(ArrayNodeDefinition $rootNode)
             ->end()
         ;
     }
+
+    private function addPhpErrorsSection(ArrayNodeDefinition $rootNode)
+    {
+        $rootNode
+            ->children()
+                ->arrayNode('php_errors')
+                    ->info('PHP errors handling configuration')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->booleanNode('log')
+                            ->info('Use the app logger instead of the PHP logger for logging PHP errors.')
+                            ->defaultValue($this->debug)
+                            ->treatNullLike($this->debug)
```

Comment:
I don't understand the default value. For me, the value does not depend on the debug flag. It should be `true` by default (most sensible value) and devs can switch it to `false` to keep BC. But then, what about having this as a temporary setting to keep BC that needs to be removed in 4.0?